requirements.txt

```diff
@@ -1 +1,8 @@
+Fiona==1.5.1
 nose==1.3.4
+numpy==1.10.1
+pandas==0.17.1
+pyshp==1.2.3
+scipy==0.16.1
+Shapely==1.2.19
+rapid_watershed_delineation==0.1.2
```

Comment:
Not inside the development VM. Otherwise, we could only test RWD after deploying a new release.